README.md

```diff
@@ -528,7 +528,7 @@ Finally, you can use the `AssistantClient`'s `GetMessages` method to retrieve th
 For illustrative purposes, you could print the messages to the console and also save any images produced by the assistant to local storage:
 
 ```csharp
-PageableCollection<ThreadMessage> messages = assistantClient.GetMessages(threadRun.ThreadId, ListOrder.OldestFirst);
+PageCollection<ThreadMessage> messages = assistantClient.GetMessages(threadRun.ThreadId, ListOrder.OldestFirst);
```

Comment:
If I'm looking at ApiView correctly, `ListOrder` is now replaced by operation/collection-specific options -- I don't see the old `ListOrder` type used here or anywhere else. Should we update the references here and retire the old class entirely?

Reply:
Sure - happy to remove that type as part of this PR.

Reply:
Per offline discussion, I was thinking this was a general list options and not something we want to keep to have a way to specify item order within a collection.  We will keep `ListOrder`!